__tests__/models/StreetParkingRulesSpec.js

```diff
@@ -29,28 +31,37 @@ describe("StreetParkingRules", () => {
     describe('when parking Monday-Saturday after 7pm', () => {
       describe('and car is on wrong side of street', () => {
         it('returns 0', () => {
-          const rules = new StreetParkingRules()
           const wednesdayEvening = new Date(2019, 3, 3, 19, 1)
           expect(rules.timeRemainingOnSide(wednesdayEvening, 'odd')).toEqual(0)
         })
       })
       describe('and car is on even side of street on a Wednesday', () => {
         it('returns minutes until switch', () => {
-          const rules = new StreetParkingRules()
           const aWednesdayMorning = new Date(2019, 3, 3, 19, 1)
           expect(rules.timeRemainingOnSide(aWednesdayMorning, 'even')).toEqual(1439 * 60)
         })
       })
       describe('and car is on odd side of street on a Tuesday', () => {
         it('returns minutes until switch', () => {
-          const rules = new StreetParkingRules()
           const aWednesdayMorning = new Date(2019, 3, 2, 19, 1)
           expect(rules.timeRemainingOnSide(aWednesdayMorning, 'odd')).toEqual(1439 * 60)
         })
       })
     })
     describe('when parking on Sunday', () => {
+      describe('and car is on odd side of the street', () => {
+        it('returns minutes until switch', () => {
+          const sundayNoon = new Date(2019, 3, 21, 12, 1)
+          expect(rules.timeRemainingOnSide(sundayNoon, 'odd')).toEqual(1859 * 60)
+        })
+      })
 
+      describe('and car is on even side of the street', () => {
+        it('returns minutes until switch', () => {
+          const sundayNoon = new Date(2019, 3, 21, 12, 1)
```

Comment:
Incorrect. 

```
Parking on the even-numbered side of the street only until 7:00 p.m. on Tuesday, Thursday and Saturday. Parking on odd-numbered side only on Sunday and until 7:00 p.m. on Monday, Wednesday and Friday
```
Plus, i would want multiple Sunday even tests, one at 12 (which would already pass), and one at 8pm (which is what you're changing), in addition to the ones on odd (12p + 7pm?)